.gitignore

```diff
@@ -127,3 +128,16 @@ ghostdriver.log
 *.swp
 /data/khan/assessmentitems.json.version
 file_map.json
+kalite/bin/*
```

Comment:
It seems you're ignoring an awful lot of files that are misplaced on your local system :) Perhaps figure out what causes that?